### ISSUE TITLE
实体识别报错torch.where(mask[i].unsqueeze(1), next_score, score)

### ISSUE THREAD

**Issue by dirac472** [liwenju0/cutword#15]
执行以下命令报错：
`from  cutword import NER
ner = NER()
res = ner.predict("机械设计制造及其自动化、测控技术与仪器、电气工程及其自动化、自动化、飞行器设计与丁程、飞行器动力工程，专业分流后到飞行器动力工程、飞行器设计与工程专业学费6760元/学年", return_words=False)`

cutword\ner.py:559: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  return torch.tensor(input_tensors), torch.tensor(seq_lens), input_lists
xxx\cutword\pytorchcrf.py:305: UserWarning: where received a uint8 condition tensor. This behavior is deprecated and will be removed in a future version of PyTorch. Use a boolean condition instead. (Triggered internally at C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\TensorCompare.cpp:493.)
  score = torch.where(mask[i].unsqueeze(1), next_score, score)
